packages/@coorpacademy-components/src/molecule/setup-slide/index.js

```diff
@@ -22,14 +23,17 @@ const SetupSlide = props => {
         return <InputCheckbox {...field} />;
       case 'image':
         return <ImageUpload {...field} />;
+      case 'splitForm':
+        return <SetupCohortItem field={field} />;
+      case 'alert':
+        return <SetupCohortItemPopin header={field.title} content={field.subtitle} />;
```

Comment:
`SetupCohortItemPopin` should be more generic: `molecule.messagePopin`

Reply:
It's Done